infra/api/app-config/prod.tf

```diff
@@ -7,6 +7,5 @@ module "prod_config" {
   domain                          = "api.simpler.grants.gov"
   database_instance_count         = 2
   database_enable_http_endpoint   = true
-  enable_v01_endpoints            = false
```

Comment:
@chouinar I wanted to leave it null since the string "False" is truthy, ya know?

```bash
$ python -c 'print(bool("False"))'
> True
```

Unless you're certain the that thing reading the env vars is casting strings to booleans "properly"!

Reply:
We use Pydantic for loading env vars / converting to Python types. It considers `false` as a false value: https://docs.pydantic.dev/2.0/usage/types/booleans/

Reply:
Added! https://github.com/HHS/simpler-grants-gov/pull/1516/commits/f32edb6ef22b36cab0f5c028ea9a090d62efc511